### ISSUE TITLE
Execution of shell script doesn't allow output/input

### ISSUE THREAD

**Issue by matkoch** [dotnet/templating#1424]
I have a postAction as follows:

    {
      "description": "Execute setup.sh.",
      "actionId": "3A7C4B45-1F5D-4A30-959A-51B88E82B5D2",
      "args": {
        "executable": "/bin/bash",
        "args": "setup.sh"
      },
      "manualInstructions": [],
      "continueOnError": false
    }

The setup.sh could be as minimal as

    echo "test"
    exit

The script is executed, but it doesn't produce any output.

Any flag I'm missing to set?

**Comment by matkoch** [comment]
I noticed that the same applies - though not very consistently - to PowerShell scripts. Only output generated from `$host.ui.PromptForChoice` and `Read-Host` is visible. All `Write-Host` calls won't appear.

**Comment by seancpeters** [comment]
@matkoch - You're not missing anything, there's no way in the shipped versions of dotnet new to get the behavior you're asking for. But it's a good idea, so I've made PR #1432 to allow stdout to go to the console. The existing code tells the invoked process to redirect stdout. With the changes in this PR, you'll be able to add a flag to the run script post action to cause stdout to not be redirected. This new flag will go in the args dictionary, like this:
```
      "args": {
        "executable": "/bin/bash",
        "args": "setup.sh",
        "redirectStandardOutput": false
      },
```

**Comment by seancpeters** [comment]
@matkoch - This PR has been merged into the release/2.1 branch, which is the active dev branch (a lot of those older, open PR's are WIP, even if not marked as such). I'm not certain when we'll next update dotnet new in the dotnet cli, hopefully not too long.

**Comment by mlorbetske** [comment]
Closing this as the change should certainly have made its way to CLI by now. If you're still not seeing the desired behavior, please re-open this. @seancpeters if you have a chance to confirm that the change is included in an earlier commit hash than the current CLI, that'd be super.
